browser/src/react-redux/App/SignupPage.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import setUser from './setUser.action'
+import axios from 'axios';
+
+const initialState = {
+  confirmPassword: '',
+  email: '',
+  password: '',
+  emailError: '',
+  generalError: '',
+  passwordError: ''
+};
+
+const mapDispatchToProps = dispatch => {
```

Comment:
It's pretty common to include these next to the export. Let's toss `mapDispatchToProps` and `mapStateToProps` down to the bottom.